Automattic/woocommerce-payments
# Add/3564 display images terminal card reader

Fixes #3564 

#### Changes proposed in this Pull Request
- add image preview for uploaded logo and icon on Card Reader settings  page.








#### Testing instructions
- go to Card Readers settings page
- upload a logo and an icon
- images should be displayed above the upload field
- click on save changes
- refresh the page - the images should be displayed also after saving the settings




*

-------------------


**Post merge**
